test/extern/unions/externUnion.chpl

```diff
@@ -0,0 +1,26 @@
+/**********************************************************
+/* This union example is also provided in
+/* $CHPL_HOME/doc/rst/language/spec/interoperability.rst
+/**********************************************************/
```

Comment:
Same nested comment issue here as in the other test.

Reply:
Sure, will fix.